package.json

```diff
@@ -48,6 +52,9 @@
     "@babel/cli": "^7.14.8",
     "@babel/preset-typescript": "^7.15.0",
     "@ember/optional-features": "^2.0.0",
+    "@glimmer/component": "^1.0.4",
+    "@glimmer/interfaces": "^0.84.1",
+    "@glimmer/reference": "^0.84.1",
```

Comment:
_someday_ we'll detangle that circular reference and then not need all these extra packages 😭